go.mod

```diff
@@ -74,3 +74,7 @@ require (
 	google.golang.org/grpc v1.31.0 // indirect
 	google.golang.org/protobuf v1.26.0 // indirect
 )
+
+require github.com/AdaLogics/go-fuzz-headers v0.0.0-20221103172237-443f56ff4ba8
```

Comment:
Can you move this one into the first (direct dependencies) block and rerun go mod tidy? Go doesn't always group them correct, and needs a bit of a nudge)